README.md

```diff
@@ -23,239 +24,233 @@
-- Monitor sensitive data flows across every component (*Data stores, internal and external APIs*)
+And many [more](https://docs.bearer.sh/reference/rules/).
 
-Curio is Open Source ([*see license*](#mortar_board-license)), and is built to be fully customizable, from creating your own rules, to adding custom code detectors up to enriching our data classifiers.
+Bearer is Open Source ([*see license*](#mortar_board-license)) and fully customizable, from creating your own rules to component detection (database, API) and data classification.
 
-Curio also powers [Bearer](https://www.bearer.com), the developer-first platform to proactively find and fix data security risks and vulnerabilities across your development lifecycle
+Bearer also powers our commercial offering, [Bearer Cloud](https://www.bearer.com), allowing security teams to scale and monitor their application security program using the same engine.
 
 ## :rocket: Getting started
 
-Discover data security risks and vulnerabilities in only a few minutes. In this guide you will install Curio, run a scan on a local project, and view the results of a summary report. Let's get started!
+Discover your most critical security risks and vulnerabilities in only a few minutes. In this guide, you will install Bearer, run a scan on a local project, and view the results. Let's get started!
 
-### Install Curio
+### Install Bearer
 
-The quickest way to install Curio is with the install script. It will auto-select the best build for your architecture. _Defaults installation to `./bin` and to the latest release version_:
+The quickest way to install Bearer is with the install script. It will auto-select the best build for your architecture. _Defaults installation to `./bin` and to the latest release version_:
 
 ```bash
-curl -sfL https://raw.githubusercontent.com/Bearer/curio/main/contrib/install.sh | sh
+curl -sfL https://raw.githubusercontent.com/Bearer/bearer/main/contrib/install.sh | sh
 ```
 
-Or, if your platform supports it, with [Homebrew](https://brew.sh/) using [Curio's official Homebrew package](https://github.com/Bearer/homebrew-curio):
-
-```bash
-brew install Bearer/curio/curio
-```
-
-#### Debian/Ubuntu
-
-```shell
-$ sudo apt-get install apt-transport-https
-$ echo "deb [trusted=yes] https://apt.fury.io/bearer/ /" | sudo tee -a /etc/apt/sources.list.d/fury.list
-$ sudo apt-get update
-$ sudo apt-get install curio
-```
-
-#### RHEL/CentOS
-
-Add repository setting
-
-```shell
-$ sudo vim /etc/yum.repos.d/fury.repo
-[fury]
-name=Gemfury Private Repo
-baseurl=https://yum.fury.io/bearer/
-enabled=1
-gpgcheck=0
-$ sudo yum -y update
-$ sudo yum -y install curio
-```
-
-[Additional installation options](#gear-additional-installation-options) are available.
+#### Other install options
+<details>
+  <summary>Homebrew</summary>
+
+  Using [Bearer's official Homebrew package](https://github.com/Bearer/homebrew-bearer):
+
+  ```bash
+  brew install Bearer/bearer/bearer
```

Comment:
@gotbadger If I'm not mistaken this won't be this path.
Can you check it as well as with the others install method? 🙏